packages/react/src/hooks/useUiElementPosition.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  useFloating,
+  UseFloatingOptions,
+  useTransitionStyles,
+} from "@floating-ui/react";
+import { useEffect } from "react";
+import { UiComponentPosition } from "../components-shared/UiComponentTypes";
+
+export function useUiElementPosition(
```

Comment:
`useFloatingUIElement`?

Reply:
I think the fact that we use FloatingUI is more of an implementation detail, so not sure if that should be reflected in the name (could say the same for `MantineSuggestionMenu`). I would just do smth like `useUiElementPositioning` or `useDefaultUiElementPositioning`.